test/parallel/test-net-dns-lookup.js

```diff
@@ -9,7 +9,7 @@ var server = net.createServer(function(client) {
 });
 
 server.listen(common.PORT, '127.0.0.1', function() {
-  net.connect(common.PORT, '127.0.0.1').on('lookup', function(err, ip, type) {
+  net.connect(common.PORT, 'localhost').on('lookup', function(err, ip, type) {
```

Comment:
Could this use [`common.localhostIPv4`](https://github.com/iojs/io.js/blob/master/test/common.js#L41)?